### DISCUSSION TITLE
瘦幾公斤才容易被注意到瘦了

### DISCUSSION THREAD

**Discussion by roma0323** [1101-GS4719-C/Python-Community-GS4719-C#1543] [General]
我覺得不管是減肥還去增重，小幅度的體重改變不容易被他人給注意到，那麼究竟要努力到什麼時候﻿，才會被人注意到你真的有瘦？﻿
以下是我後來找到的資料
﻿
有研究使用一款神奇的電腦程式﻿，先模擬出人臉在不同體重下會產生的變化﻿
再調查人們看了這些模擬結果後的感想﻿
然後發現，一旦減到4公斤這個里程碑就能被多數人發現你瘦了﻿
﻿
此外還進一步發現﻿
當瘦到6至8公斤時，人的顏值，還會再提升一個檔次：﻿
旁人不僅會說你瘦了，還會開始驚呼﻿
天啊你變帥了 / 變美了！

**Comment by Miinteaa** [comment]
感覺瘦多少公斤能被看出來，每個人應該會有不太一樣的結果，因為肌肉跟脂肪的比重不太一樣，如果今天是瘦了4公斤的脂肪，外觀上可能會落差很大，且每個人首先胖的位置都不一樣，有些人先胖肚子，再由衣服遮蔽掉，感覺不容易被察覺

**Comment by zhiron544** [comment]
我覺得這篇文章很有意思，也很有幫助。我自己也在減肥中，看了這篇文章後，覺得有了一些方向和期待。不過，我也同意每個人的情況不同，不能一概而論。我想最重要的是要找到適合自己的方法和節奏，並且享受過程。減肥不只是為了別人的眼光，更是為了自己的健康和快樂。

**Comment by Alex11392** [comment]
好有趣的分享，我自己不管吃多吃少體重變化都不會超過2公斤，所以完全沒有這樣的經驗

**Comment by 110302001** [comment]
我覺得這和公斤數沒有直接的關係耶，有些人不管怎麼胖就是不會胖到臉，也有些人不管身體怎麼纖瘦臉都還是肉肉的，所以這些人就都比較難察覺到瘦了還沒是沒瘦

**Comment by ThanatosJun** [comment]
我認為瘦跟胖的界定如果只以體重公斤算的話並不恰當，真正診斷的標準應當是肌肉的密度，如果一個外表看似纖細的人有結實的肌肉，那他的體重也有可能比外型更大的人還要重。我認為胖瘦無法完全以二分法區分，他們類似光譜的兩端。

**Comment by THH1118** [comment]
體感到瘦應該跟人的比例有關，但我覺得只是要看有沒有瘦的話看自己穿褲子的鬆緊最有效哈哈哈

**Comment by Chengbosco** [comment]
我覺得對自己瘦了多少實在沒有感覺，唯一的感覺方法就是褲子的鬆緊WWWWW，倒是別人很容易就看出來了。

**Comment by 111426027** [comment]
我覺得減脂比較容易看出來變瘦，因為有時候公斤數減少可能只是攝取的水分或當天吃的比較少造成的。

**Comment by Sherry9106** [comment]
我們在觀察一個人是否變瘦的時候好像大部分都是看臉？除非身體有很大的變化，不然應該很難看出來，網路上還有一些瘦臉操的影片，讓你在別人眼中更加苗條。

**Comment by spermwhale464** [comment]
要看瘦哪吧，瘦臉超快被看到，吸cannabis一下就被看出來了

**Comment by poiu6fi** [comment]
我最近胖了好幾公斤，不過都是胖在肚子上，真的好險不是胖在臉上

**Comment by Tchang323** [comment]
我感覺要看個人的體質耶，有些人可能瘦2~3公斤，就會有明顯差距，比如說臉型。

**Comment by Tchang323** [comment]
如果有想特別瘦身哪個地方而針對去做運動，那可能減重的過程，不用減太多，外表會有明顯改善

**Comment by yue083139** [comment]
這篇文章我個人的經驗是蠻有感覺的，自從上大學之後，每次回家家人都說我好像又變瘦了，原本我是不在意的，覺得只是太久沒看到人產生的錯覺，直到之前寒假突然跑去量體重發現瘦了4公斤左右再去跟高中的照片對比，真的想不到在短時間內一個人能有這麼顯著的變化，感覺之後如果再瘦一點下去或許就能實驗看看6公斤長相會不會變動的事情了（雖然要再瘦下去感覺也不太容易了）
